eox_nelp/settings/common.py

```diff
@@ -82,3 +82,7 @@ def plugin_settings(settings):
     except AttributeError:
         # We must find a way to register this error
         pass
+    # Ensure EOX_NELP is the last plugin installed app for ready run.
+    if EOX_NELP_MODEL_APP in settings.INSTALLED_APPS and settings.INSTALLED_APPS[-1] != EOX_NELP_MODEL_APP:
```

Comment:
could you move this logic to the settings file(nelc-dev-kit/plugins/extra_env_patches.py and  manifest-sagar/plugins) ?? I make this suggestion because you're fixing the app position and if we need to change that later, this implementation makes harder to modify that order

Reply:
> could you move this logic to the settings file(nelc-dev-kit/plugins/extra_env_patches.py and manifest-sagar/plugins) ?? I make this suggestion because you're fixing the app position and if we need to change that later, this implementation makes harder to modify that order

Yes I could, also in the plugin you could reorder. As you see custom_reg_form is appended using the env patches. Anyway, if you think is better the fix there I could do it.